tests/certification/bindings/azure/blobstorage/blobstorage_test.go

```diff
@@ -0,0 +1,731 @@
+		assert.NoError(t, invokeCreateErr)
+
+		blobName := out.Metadata["blobName"]
+		res, _ := getBlobRequest(ctx, client, blobName, false)
+		oldString := string(res.Data)
+
+		input2 := "some other example content"
+		dataBytes2 := []byte(input2)
+
+		invokeCreateMetadata2 := map[string]string{
+			"blobName":    blobName,
+			"contentType": "text/plain",
+		}
+
+		invokeCreateRequest2 := &daprsdk.InvokeBindingRequest{
+			Name:      "azure-blobstorage-output",
+			Operation: "create",
+			Data:      dataBytes2,
+			Metadata:  invokeCreateMetadata2,
+		}
+		_, invokeCreateErr2 := client.InvokeBinding(ctx, invokeCreateRequest2)
+
+		assert.NoError(t, invokeCreateErr2)
+
+		res2, _ := getBlobRequest(ctx, client, blobName, false)
+		newString := string(res2.Data)
+
+		assert.NotEqual(t, oldString, newString)
+		assert.Equal(t, newString, input2)
+
+		// cleanup.
+		out, invokeDeleteErr := deleteBlobRequest(ctx, client, blobName, "")
+		assert.NoError(t, invokeDeleteErr)
+		assert.Empty(t, out.Data)
+
+		// confirm the deletion.
+		_, invokeSecondGetErr := getBlobRequest(ctx, client, blobName, false)
+		assert.Error(t, invokeSecondGetErr)
+		assert.Contains(t, invokeSecondGetErr.Error(), "ServiceCode=BlobNotFound")
+
+		// deleting the key again should fail.
+		_, invokeDeleteErr2 := deleteBlobRequest(ctx, client, blobName, "")
+		assert.Error(t, invokeDeleteErr2)
+		assert.Contains(t, invokeDeleteErr2.Error(), "ServiceCode=BlobNotFound")
+
+		return nil
+	}
+
+	testCreateBlobInvalidContentHash := func(ctx flow.Context) error {
+		// verifies that the content hash is validated.
+		client, clientErr := daprsdk.NewClientWithPort(fmt.Sprint(currentGRPCPort))
+		if clientErr != nil {
+			panic(clientErr)
+		}
+		defer client.Close()
+
+		input := "some example content"
+		dataBytes := []byte(input)
+		wrongBytesForContentHash := []byte("wrong content to hash")
+		h := md5.New() // nolint:gosec
+		h.Write(wrongBytesForContentHash)
+		md5HashBase64 := base64.StdEncoding.EncodeToString(h.Sum(nil))
+
+		invokeCreateMetadata := map[string]string{
+			"contentMD5": md5HashBase64,
+		}
+
+		invokeCreateRequest := &daprsdk.InvokeBindingRequest{
+			Name:      "azure-blobstorage-output",
+			Operation: "create",
+			Data:      dataBytes,
+			Metadata:  invokeCreateMetadata,
+		}
+
+		_, invokeCreateErr := client.InvokeBinding(ctx, invokeCreateRequest)
+		assert.Error(t, invokeCreateErr)
+		assert.Contains(t, invokeCreateErr.Error(), "ServiceCode=Md5Mismatch")
+
+		return nil
+	}
+
+	testCreateBlobFromFile := func(isBase64 bool) func(ctx flow.Context) error {
+		// uploads a blob from a file and optionally verifies the automatic base64 decoding option.
+		return func(ctx flow.Context) error {
+			client, clientErr := daprsdk.NewClientWithPort(fmt.Sprint(currentGRPCPort))
+			if clientErr != nil {
+				panic(clientErr)
+			}
+			defer client.Close()
+
+			dataBytes, dataErr := os.ReadFile("dapr.svg")
```

Comment:
You don't need an external file for this. You can simply save a small byte array. Also, SVG seems to be a text file.